packages/dialtone-vue2/recipes/leftbar/general_row/general_row.vue

```diff
@@ -410,4 +442,16 @@ export default {
 
 <style lang="less" scoped>
 @import "../style/leftbar_row.less";
+.unread-count-badge {
+  border-radius: 50vh 0 0 50vh;
+}
+
+.unread-mention-count-badge {
+  border-radius: 0 50vh 50vh 0;
+  background-color: var(--color-purple-500, #3A1D95);
+}
+
+.unread-mention-count-only-badge {
+  background-color: var(--color-purple-500, #3A1D95);
+}
```

Comment:
put these directly into leftbar_row.less, with the same naming convention it uses in there.

ex/ `dt-leftbar-row__unread-mention-count-badge`